examples/tiled_ipywidget.ipynb

```diff
@@ -0,0 +1,263 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tiled_file_widget.value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot the segmentation result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# With the selected job_name in the dropdown, you get the scan_uri\n",
+    "# Get the mask uri and plot the classes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.patches as mpatches\n",
+    "from matplotlib.colors import ListedColormap"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_seg_mask(results_uri, slice_idx, axs):\n",
+    "    result_client = from_uri(results_uri)\n",
+    "    result_client_metadata = result_client.metadata\n",
+    "    mask_uri = result_client_metadata[\"mask_uri\"]\n",
+    "    mask_uri = mask_uri[:-5]\n",
+    "\n",
+    "    seg_result = result_client[slice_idx]\n",
+    "\n",
+    "    # Extract class information\n",
+    "    mask_client = from_uri(mask_uri)\n",
+    "    mask_client_metadata = mask_client.metadata\n",
+    "    class_meta_data = mask_client_metadata[\"classes\"]\n",
+    "    max_class_id = len(class_meta_data.keys()) - 1\n",
+    "\n",
+    "    colors = [\n",
+    "        annotation_class[\"color\"] for _, annotation_class in class_meta_data.items()\n",
+    "    ]\n",
+    "    labels = [\n",
+    "        annotation_class[\"label\"] for _, annotation_class in class_meta_data.items()\n",
+    "    ]\n",
+    "    # Add color for unlabeled pixels\n",
+    "    colors = [\"#D3D3D3\"] + colors\n",
+    "    labels = [\"Unlabeled\"] + labels\n",
+    "\n",
+    "    axs.imshow(\n",
+    "        seg_result,\n",
+    "        cmap=ListedColormap(colors),\n",
+    "        vmin=-1.5,\n",
+    "        vmax=max_class_id + 0.5,\n",
+    "    )\n",
+    "    axs.set_title(mask_client_metadata[\"project_name\"] + \", slice: \" + str(slice_idx))\n",
+    "\n",
+    "    # create a patch for every color\n",
+    "    patches = [\n",
+    "        mpatches.Patch(color=colors[i], label=labels[i]) for i in range(len(labels))\n",
+    "    ]\n",
+    "    # Plot legend below the image\n",
+    "    axs.legend(\n",
+    "        handles=patches, loc=\"upper center\", bbox_to_anchor=(0.5, -0.075), ncol=3\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "selected_scan_uri = tiled_file_widget.value\n",
+    "\n",
+    "fig = plt.figure(layout=\"constrained\")\n",
+    "plot_seg_mask(selected_scan_uri, 0, fig.gca())\n",
```

Comment:
This also works nicely. A point of improvement could be to have a function that creates the overall plot and that automatically detects how many subplots are needed - depending on how many frames we have, so that the function is called once only for the set of segmentation results that were selected.